web/templates/appShell.html

```diff
@@ -1,153 +1,173 @@
+                              </svg>
+                              API Keys
+                          </a>
+                      </li>
+                  </ul>
+              </li>
+              <li class="profile-item">
+                  <a href="#" class="profile-link">
+                      <img
+                          class="profile-img"
+                          src="/static/images/profile/user.svg"
+                          alt="Profile Image"
+                      />
+                      <span class="profile-span">Your profile</span>
+                      <span aria-hidden="true">Your Profile</span>
+                  </a>
+              </li>
           </ul>
-        </li>
-        <li class="profile-item">
-          <a href="#" class="profile-link">
-            <img
-              class="profile-img"
-              src="/static/images/profile/user.svg"
-              alt="Profile Image"
-            />
-            <span class="profile-span">Your profile</span>
-            <span aria-hidden="true">Your Profile</span>
-          </a>
-        </li>
-      </ul>
-    </nav>
+      </nav>
   </div>
 </div>
 <div class="appShell-header-mobile">
   <button type="button" class="buttoner">
-    <span class="spanner">Open sidebar</span
-    ><svg
-      xmlns="http://www.w3.org/2000/svg"
-      fill="none"
-      viewBox="0 0 24 24"
-      stroke-width="1.5"
-      stroke="currentColor"
-      aria-hidden="true"
-      class="svger"
-    >
-      <path
-        stroke-linecap="round"
-        stroke-linejoin="round"
-        d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
-      ></path>
-    </svg>
+      <span class="spanner">Open sidebar</span
+      ><svg
+          xmlns="http://www.w3.org/2000/svg"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke-width="1.5"
+          stroke="currentColor"
+          aria-hidden="true"
+          class="svger"
+      >
+          <path
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
+          ></path>
+      </svg>
   </button>
   <div class="diver">Integrand</div>
   <a href="#">
-    <span class="spanner">Your profile</span>
-    <img
-      class="mobile-profile-image"
-      src="/static/images/profile/user.svg"
-      alt="Profile Image"
-    />
+      <span class="spanner">Your profile</span>
+      <img
+          class="mobile-profile-image"
+          src="/static/images/profile/user.svg"
+          alt="Profile Image"
+      />
   </a>
 </div>
 <script>
   //TODO: Fix this junk
   function selectMenuItem() {
-    let path = window.location.pathname;
-    let selected_item = "";
-    if (path.includes("topics")) {
-      selected_item = "2";
-    } else if (path.includes("connectors")) {
-      selected_item = "3";
-    } else {
-      selected_item = "1";
-    }
-    let selected_classname = "selected-item";
+      let path = window.location.pathname;
```

Comment:
To get the highlighting on click, you need to add a condition here for you keys api. If you used the same class names for everything, this should happen automatically once your branch condition is applied.

Reply:
This one is fixed with the new commit.